### PR TITLE
Refs #33491 - make the hostname bold in delete modal

### PR DESCRIPTION
**Pull request by amirfefer**
Screen Shot 2021-10-12 at 16 32 37

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js [hunk 1]
```diff
@@ -53,7 +45,18 @@ export const deleteHost = (
...
+      message: (
+        <FormattedMessage
+          id="delete-host"
+          values={{
+            host: <b>{hostName}</b>,
+            cascade: cascadeMessage(),
+          }}
+          defaultMessage={__(
+            'Are you sure you want to delete host {host}? This action is irreversible. {cascade}'
+          )}
```
Comment (tbrisker) [inline webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:57]: I'm not sure but I think you need to use `sprintf` to properly interpolate the values into the translated string.

Comment (amirfefer) [reply webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:57]: doesn't `react-intl` formatting interpolate the strings? is there a quick way to test it out?

Comment (ezr-ondrej) [reply webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:57]: It works just fine :)

**Review by ezr-ondrej — APPROVED** [review]
Works well, thanks @amirfefer ! :+1:

**Pull request merged**